index.html

```diff
@@ -1,10 +1,521 @@
+            itself would
+            become imbued with a mysterious latent energy.</p>
+        <p>Inserted into Einstein’s equations, the latent energy would act as a kind of antigravity, and the
+            universe would
+            blow itself up. Since it was space itself supplying the repulsive force, the more space was created,
+            the harder it
+            pushed apart.</p>
+        <p>What would become our observable universe mushroomed in size at least a trillion trillionfold — from a
+            submicroscopic
+            speck of primordial energy to the size of a grapefruit — in less than a cosmic eye-blink.</p>
+        <p>Almost as quickly, this pulse would subside, relaxing into ordinary particles and radiation. All of
+            normal cosmic
+            history was still ahead, resulting in today’s observable universe, a patch of sky and stars billions of
+            light-years
+            across. “It’s often said that there is no such thing as a free lunch,” Dr. Guth likes to say, “but the
+            universe might
+            be the ultimate free lunch.”</p>
+        <p>Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision
+            suggest that
+            inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey
+            expansion, the rest
+            of the cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the
+            multiverse.</p>
+        <p>So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any
+            deeper into the past.</p>
+        <p>We might never know what happened before inflation, at the very beginning, because inflation erases
+            everything that came
+            before it. All the chaos and randomness of the primordial moment are swept away, forever out of our
+            view.</p>
+        <p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part
+            of the team,
+            “you wind up at inflation.”</p>
+        <p>A version of this article appears in print on March 18, 2014, on Page A1 of the New York edition with
+            the headline:
+            Space Ripples Reveal Big Bang’s Smoking Gun.<a href=""> Order Reprints </a>|<a href=""> Today’s Paper
+            </a>|<a href="">
+                Subscribe</a></p>
+
+        <div class="btm-btns">
+            <button class="s-icons"><i class="material-icons">reply</i></button>
+            <button class="s-icons"><i class="material-icons">mail</i></button>
+            <button class="s-icons" style="font-size: 23px">t</button>
+            <button class="s-icons" style="font-size: 23px">f</button>
+            <button>READ 615 COMMENTS</button>
+            <br /><br /><br>
```

Comment:
Don't add multiple `br` elements for spacing. If you want to add spacing do it through CSS